src/components/common/ClassroomCard.jsx

```diff
@@ -30,12 +48,30 @@ const ClassroomCard = ({ isSelected, classroomDetails, classroomNo, onCardSelect
 						{/** TODO: Need to iterate through classroomDetails here using map method and render key value pairs. */}
 						<StyledCardInfo>
 							<StyledCardInfoTitle>Total</StyledCardInfoTitle>
-							<StyledCardInfoValue>35</StyledCardInfoValue>
-						</StyledCardInfo>
-						<StyledCardInfo>
-							<StyledCardInfoTitle>Current Strength</StyledCardInfoTitle>
-							<StyledCardInfoValue>30</StyledCardInfoValue>
+							<StyledCardInfoValue id="totalStrength">
+								{totalStrength}
+							</StyledCardInfoValue>
 						</StyledCardInfo>
+						{minSeatDifference ? (
+							<StyledCardInfo className="danger-color">
```

Comment:
Do not use className on StyledComponents
```suggestion
							<StyledCardInfo currentStrength={currentStrength} totalStrength={totalStrength}>
```

Reply:
resolved